configs/openSUSE/pam-modules.toml

```diff
@@ -422,9 +422,9 @@ nodigests = [
 ]
 
 [[FileDigestGroup]]
-packages = ["pam_kwallet", "pam_kwallet6"]
+packages = ["pam_kwallet6"]
 type = "pam"
-note = "imported from rpmlint1 PAMModules.WhiteList"
+note = "unlocks the kwallet during the session phase of PAM login"
 bugs = ["bsc#993806", "bsc#1217183"]
 nodigests = [
     "glob:*/security/pam_kwallet5.so",
```

Comment:
Is the kwallet 6 PAM module still called pam_kwallet5.so even after the full KDE 6 release?

Reply:
yes it's still named like this, although I mentioned it in the AUDIT bug